dwave/preprocessing/presolve/cypresolve.pyx

```diff
@@ -60,12 +78,20 @@ cdef class cyPresolver:
 
     def apply(self):
         """Apply any loaded presolve techniques to the held constrained quadratic model."""
-        self.cpppresolver.apply()
+        try:
+            self.cpppresolver.normalize()
```

Comment:
Have you considered splitting out the `normalize()` and `presolve()` methods in the python interface as well?

Reply:
Yes, planning to do that in the next PR. Just didn't want to combine too many features here.